Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.17 as builder
+FROM golang:1.18.4 as builder
```

Comment:
Shouldn't we leave this at `golang:1.18` so it picks up the latest patch release of Go? Otherwise we might need to update all these references often.

Reply:
Agreed, will fix it.